docs/tutorial.rst

```diff
@@ -0,0 +1,1605 @@
+reactions and compounds. This feature can be useful, for example, if the user
+wants to name different reaction files based on the subsystem designations or
+cellular compartments, or if the user wants to separate the temporary reactions
+that are used to fill reaction gaps from the main model. An example of how a
+user might designate multiple reaction files is below. This file can be found
+in the additional files folder in the file complex_model.yaml.
+
+.. code-block:: shell
+
+    name: Ecoli_core_model
+    biomass: R_Biomass_Ecoli_core_w_GAM
+    default_flux_limit: 1000.0
+    model:
+    - include: core_model_definition.tsv
+    compounds:
+    - include: compounds.yaml
+    reactions:
+    - include: reactions/cytoplasm.yaml
+    - include: reactions/periplasm.yaml
+    - include: reactions/transporters.yaml
+    - include: reactions/extracellular.yaml
+    media:
+    - include: medium.yaml
+    limits:
+    - include: limits.yaml
+
+
+As can be see here the modeler chose to distribute their reaction database
+files into different files representing various cellular compartments and roles.
+This organization can be customized to suit the workflow of each individual
+user.
+
+There are also situations where the user may wish to designate only a subset
+of the reaction database in a metabolic simulation. In these situations it is
+possible to use a model definition file to identify which subset of reactions
+will be used from the larger database. The model definition file is simply a
+list of reaction IDs that will be included in the simulation.
+
+An example of how to include a model definition file can be found below.
+
+.. code-block:: shell
+
+    name: Ecoli_core_model
+    biomass: R_Biomass_Ecoli_core_w_GAM
+    default_flux_limit: 1000.0
+    model:
+    - include: subset.tsv
+    compounds:
+    - include: compounds.yaml
+    reactions:
+    - include: reactions.yaml
+    media:
+    - include: medium.yaml
+    limits:
+    - include: limits.yaml
+
+``Note``: When the model definition file is not identified, `PSAMM` will include
+the entire reaction database in the model. However, when it is identified,
+`PSAMM` will only include the reactions that are listed in the model definition
+file in the model. This design can be useful when a user wants to make targeted
+tests on a subset of the model or when the user wants to include a larger
+database for use with the gapfill function.
+
+A) Reactions
+~~~~~~~~~~~~~
+
+The reactions.yaml file is where the reaction information is stored in the
+model. A sample of this kind of file can be seen below:
+
+.. code-block:: shell
+
+    - id: R_ACALDt
+    name: acetaldehyde reversible transport
+    equation: '|M_acald_e[C_e]| <=> |M_acald_c[C_c]|'
+    subsystem: Transport, Extracellular
+    - id: R_ACKr
+    name: acetate kinase
+    equation: '|M_ac_c[C_c]| + |M_atp_c[C_c]| <=> |M_actp_c[C_c]| + |M_adp_c[C_c]|'
+    subsystem: Pyruvate Metabolism
```

Comment:
The keys following the id should be indented with two spaces to line up with `id`, for example
``` yaml
- id: R_ACALDt
  name: acetaldehyde reversible transport
  equation: '|M_acald_e[C_e]| <=> |M_acald_c[C_c]|'
  subsystem: Transport, Extracellular
- id: R_ACKr
[...]
```